crates/biome_string_case/src/lib.rs

```diff
@@ -399,7 +403,7 @@ impl Iterator for CasesIterator {
         (0, Some(6))
     }
 }
-impl std::iter::FusedIterator for CasesIterator {}
+impl core::iter::FusedIterator for CasesIterator {}
```

Comment:
Why the change?

Reply:
No particular reason. I think using `core` is better than `std`. However, this should be moved to a separate PR. I will remove the change.